lib/anyway/settings.rb

```diff
@@ -97,7 +97,7 @@ def matching_env?(env)
     end
 
     # By default, use local files only in development (that's the purpose if the local files)
-    self.use_local_files = (ENV["RACK_ENV"] == "development" || ENV["RAILS_ENV"] == "development")
+    self.use_local_files = (ENV["RACK_ENV"] == "development" || ENV["RAILS_ENV"] == "development" || Rails&.env == 'development')
```

Comment:
We should move this check to Rails part of the library, somewhere here: https://github.com/palkan/anyway_config/blob/c2ae118068b4edeaf242061eee06096f7976a21a/lib/anyway/rails/settings.rb#L78

Smth like:

```ruby
self.use_local_files ||= Rails.env.development?
```

Reply:
updated, thanks!